release notes/v0.26.0.md

```diff
@@ -0,0 +1,122 @@
+k6 v0.26.0 is here! :tada:
+
+This release contains mostly bug fixes, though it also has several new features and enhancements! They include a new JS compatibility mode option, exporting the end-of-test summary to a JSON report file, speedups to the InfluxDB and JSON outputs, a brand new CSV output, multiple layered HTTP response body decompression, being able to use `console` in the init context, a new optional column in the summary, and Docker improvements!
+
+
+Thanks to  @Sirozha1337, @openmohan, @MMartyn, @KajdeMunter, @dmitrytokarev and @dimatock for contributing to this release!
+
+## New features and enhancements!
+
+### A new JavaScript compatibility mode option (#1206)
+
+This adds a way to disable the automatic script transformation by [Babel](https://babeljs.io/) (v6.4.2) and [core-js (v2)](https://github.com/zloirock/core-js) polyfills bundled in k6. With the new `base` compatibility mode, k6 will instead rely only on the [goja](https://github.com/dop251/goja/) runtime and what is built into k6.
+This can be configured through the new `--compatibility-mode` CLI flag and the `K6_COMPATIBILITY_MODE` environment variable. The possible values currently are:
+
+- `extended`: this is the default and current compatibility mode, which uses Babel and core.js to achieve ES6+ compatibility.
+- `base`: an optional mode that disables loading of Babel and core.js, running scripts with only goja's native ES5.1+ compatibility. If the test scripts don't require ES6 compatibility (e.g. they were previously transformed by Babel), this option can be used to reduce RAM usage during test runs.
+
+More info what this means can be found in the [documentation](https://docs.k6.io/docs/javascript-compatibility-mode).
+
+Our benchmarks show considerable drop in memory usage - around 80% for simple scripts, and around 50% in the case of 2MB script with a lot of static data in it. The CPU usage is mostly unchanged, except that k6 initializes a test run a lot faster. All of those benefits will be most noticeable if k6 is used with big number of VUs (1k+). More performance comparison can be found in #1167.
+
+### JSON export of the end-of-test summary report (#1168)
+
+This returns (from the very early days of k6) the ability to output the data from the end of test summary in a machine readable JSON file.
+This report can be enabled by the `--summary-export <file_path>` CLI flag. The resulting JSON file will include data for all test metrics, checks and thresholds.
```

Comment:
Should we mention the environment variable `K6_SUMMARY_EXPORT`, too?

Reply:
```suggestion
This report can be enabled by the `--summary-export <file_path>` CLI flag or the `K6_SUMMARY_EXPORT` environment variable. The resulting JSON file will include data for all test metrics, checks and thresholds.
```